websites/M/Maliki/presence.ts

```diff
@@ -0,0 +1,110 @@
+const presence = new Presence({
+		clientId: "990204576189808670",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+interface songPlayingInformations {
```

Comment:
```suggestion
interface SongPlayingData {
```

Reply:
Rename in 478d0d23d9b6c22f270fba74fa3b89e24676d745

Reply:
I changed the casing on purpose, it should use the name I suggested.

Reply:
I don't undertstand what you mean. I've changed the name to the one who've suggested ?

Reply:
Oh okay, I'm dumb, i've didn't see the uppercase S

Reply:
Right rename in a65fb62f8172dd2e9827123bae9036b2484f372c